### DISCUSSION TITLE
Double voice reproduced by MiniDexed controlled via Midi from Casio CDP 100

### DISCUSSION THREAD

**Discussion by eziosoma** [probonopd/MiniDexed#400] [Q&A] [answered]
Hello,
"Houston we have a problem!".
Me and my son have assembled a MiniDexed with LCD display, Midi input, DAC and rotary encoder.
We have connected the mounted device via  midi to the Casio CDP 100 electronic piano.
What happens is that if we select the Voice through the Casio CDP 100 buttons, the voice is correctly selected. If we select the voice trough the rotary encoder the voices that are reproduced are two: the last one selected on the Casio keyboard and the one shown by the LCD.

This is quite weird. Is there someone that can help us? Could it be a configuration problem?

Many thanks in advance for your help!

**Comment by Banana71** [comment, marked as answer]
Hi @eziosoma,

I think you have the same problem as in this post #384
Peter

**Reply by eziosoma** [reply]
Yes indeed!
I solved the problem. Thankyou very much Banana71

**Reply by Banana71** [reply]
have fun making music together,
